server/views/components/interview/index.njk

```diff
@@ -0,0 +1,12 @@
+<div class="interview">
+  {% for qAndA in interview %}
+    <p class="interview__question">
+      <span class="interview__participant">{{ qAndA.question.participant }}</span>
```

Comment:
Arguably too - having the person's name in the `p` doesn't semantically make that much sense, as paragraph is 
a self contained bit of text.